src/components/LegacyTooltip/LegacyTooltip.tsx

```diff
@@ -150,6 +151,7 @@ export const LegacyTooltip = ({
     children,
     position = TooltipPosition.Bottom,
     alignment = TooltipAlignment.Middle,
+    withStrongBorder,
```

Comment:
Can we set a default value of `false` to be explicit and avoid any possible type corrosion qualifiers?  e.g. `withStrongBorder === false`